package.json

```diff
@@ -3,13 +3,15 @@
   "version": "0.6.0",
   "description": "Additional Jest matchers",
   "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "files": [
     "dist",
     "README.md",
     "LICENSE"
   ],
   "scripts": {
     "build": "babel src -d dist --ignore *.test.js",
+    "postbuild": "cp types/index.d.ts dist/index.d.ts",
```

Comment:
I think this can be deleted by updating the `files` property above to be:

```json
"files": [
  "dist",
  "README.md",
  "LICENSE",
  "types/index.d.ts"
 ]
```